### PR TITLE
Return branding in bootstrap information. Closes #160.

### LINKED ISSUE #160: Provide branding information in bootstrap API endpoint

### PR DESCRIPTION
**Pull request by troeger**

### CODE REVIEW

File: kubeportal/api/views/info.py [hunk 1]
```diff
@@ -25,8 +25,7 @@ class InfoDetailView(GenericAPIView):
              'k8s_pod_count': api.get_number_of_pods,
              'k8s_volume_count': api.get_number_of_volumes,
              'portal_user_count': get_user_count,
-             'portal_version': get_kubeportal_version,
-             'k8s_cluster_name': get_cluster_name,
```
Comment (DenizCankurtaran) [inline kubeportal/api/views/info.py:29]: I know that having the cluster_name displayed in the info section is redundant, but isn't it also a valid info to display?

**Review by DenizCankurtaran — APPROVED** [review]
looks great

File: kubeportal/api/views/info.py (hunk 1 above)
Comment (troeger) [reply kubeportal/api/views/info.py:29]: I came to the conclusion that "cluster name" and "portal branding" is semantically something different, meanwhile. Kubeportal is not always about K8S access. I therefore vote for leaving out the idea of a "cluster name" for the moment.

**Review by kat-hi — APPROVED** [review]
toll

**Pull request merged**